src/App.js

```diff
@@ -4,29 +4,31 @@ import Header from "./Header";
 import Main from "./Main";
 import Footer from "./Footer";
 import allBeasts from './data.json';
-import SelectedBeast from './SelectedBeast.js';
+import SelectedBeast from './SelectedBeast';
 
 class App extends React.Component {
-constructor(props){
-  super(props);
-  this.state = {
-    allBeasts: allBeasts,
-    show: false
+  constructor(props) {
+    super(props);
+    this.state = {
+      allBeasts: allBeasts,
+      show: false,
+      selectedBeast: {}
+    }
```

Comment:
From my surveying the code, it looks like you're using very consistent naming conventions, which is great! That will serve you really well. It helped me a great deal in looking through your code and tracking how data is being passed down.